app/views/teacher_interface/application_forms/show/_declined.html.erb

```diff
@@ -14,7 +14,14 @@
   <% else %>
     <ul class="govuk-list govuk-list--bullet">
       <% reasons.each do |reason| %>
-        <li><%= simple_format reason %></li>
+        <% parts = reason.split("\n\n") %>
+        <% decline_reason = parts[0] %>
+        <% assessor_note = parts[1] %>
```

Comment:
This approach doesn't work if the assessor adds multiple new lines to their note, in that case we'll only see the first paragraph of the assessor note. It also feels quite brittle, if we were to add a decline reason with newlines in it would break too. [The two strings are combined with the newlines in the view object](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/0e1178b4b394eba490a0b9d342d10aff58eb84e0/app/view_objects/teacher_interface/application_form_view_object.rb#L256) and then we're splitting them out again here, we could instead keep the two fields separate until they're rendered in the view.

I think a better alternative would be to change the structure of the return value of the [`declined_reasons`](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/0e1178b4b394eba490a0b9d342d10aff58eb84e0/app/view_objects/teacher_interface/application_form_view_object.rb#L62) to separate the assessor feedback from the decline reason. Currently the return value is something like:

```rb
{
  "section title" => [
    "decline reason \n\n assessor note"
  ]
}
```

But the return value could be changed to:

```rb
{
  "section title" => [
    {
      decline_reason: "...",
      assessor_note: "...",
  ]
}
```

This refactor will require some changes wherever `declined_reasons` is used so I would suggest you do the refactor first in a separate PR without changing any of the content (this can be deployed without needing a product review) and then make the content changes in this PR, this'll mean the two PRs are smaller and easier to review and understand.